rockcraft/plugins/python_plugin.py

```diff
@@ -75,7 +75,7 @@ class PythonPlugin(python_plugin.PythonPlugin):
     @override
     def _should_remove_symlinks(self) -> bool:
         """Overridden because for ubuntu bases we must always remove the symlinks."""
-        return self._part_info.base != "bare"
+        return bool(self._part_info.base != "bare")
```

Comment:
this is an odd one, what was mypy complaining about?

Reply:
https://github.com/python/mypy/issues/5697#issuecomment-425738430